app/src/main/java/io/intrepid/dictionary/DictionaryContract.kt

```diff
@@ -0,0 +1,9 @@
+package io.intrepid.dictionary
+
+interface DictionaryView {
+    fun onDefinitionReceived(definition: String)
+}
+
+interface DictionaryPresenter {
+    fun getDefinition(word: CharSequence)
```

Comment:
so my thinking for MVP/MVVM is generally -- the only role of the view is to report what's happening to the presenter, and the presenter issues commands/imperatives to the view; by that logic, the methods would be named something akin to DictionaryPresenter#onDefineClick and DictionaryView#displayDefinition. in this particular case, the difference is subtle, but the dynamic is inverted; the presenter is notified of what happens (onX methods) and the view is told, within reason, exactly what to do